st2client/tests/unit/test_action_alias.py

```diff
@@ -21,9 +21,10 @@
 from tests import base
 
 from st2client import shell
+from st2client import models
```

Comment:
Following the pattern in #5333, I would switch this to use explicit imports
```suggestion
from st2client.models.core import ResourceManager
from st2client.models.action import Execution
```